tornado-drivers/drivers-common/src/main/java/uk/ac/manchester/tornado/drivers/common/compiler/phases/loops/TornadoPartialLoopUnroll.java

```diff
@@ -73,12 +69,8 @@ private static int getUnrollFactor() {
         return (isPowerOfTwo(Tornado.UNROLL_FACTOR) && Tornado.UNROLL_FACTOR <= 32) ? Tornado.UNROLL_FACTOR : LOOP_UNROLL_FACTOR_DEFAULT;
     }
 
-    private static int getUpperGraphLimit(int initialGraphNodeCount, StructuredGraph graph) {
-        return (initialGraphNodeCount + GraalOptions.MaximumDesiredSize.getValue(graph.getOptions()) * 2);
-    }
-
-    public static LoopPolicies createLoopPolicies() {
-        return new DefaultLoopPolicies();
+    private static int getUpperGraphLimit(int initialGraphNodeCount) {
+        return (initialGraphNodeCount + (20000 * 2));
```

Comment:
Why these numbers? 

Add a constant with the whole computation.

Reply:
fixed